### ISSUE TITLE
Possible bug for Voronoi anlaysis after box is flipped upon shearing

### ISSUE THREAD

**Issue by akohlmey** [lammps/lammps#77]
From lammps-users:

Dear Steve, 

There might be a bug for the voronoi analysis after the simulation box is flipped upon shearing. I noticed there is an early post for discussing the voronoi analysis of a triclinic box

http://lammps.sandia.gov/threads/msg60447.html 

I used the latest version (14May16). It is no problem for the normal shearing with this version. However, when I do a severe shearing simulation that allows box flipping. I found after the flipping, only a part of the atoms have the values from the voronoi analysis, such as atomic volume, number of faces, but the rest are all zero. So it seems that there is still a bug in dealing with the triclinic box. I am trying to debug it but still not making it all right...

Best regards,
Suzhi

**Comment by akohlmey** [comment]
Dear Axel, 

Thanks. My case is in a relatively large size. Here I tested a simple case in a small system (see below).  The values from voronoi analysis have a large change after box flipping (after step 125 000). For example, the volume of Voronoi cell (c_1[1]) of some atoms can even go to be 76 A^3 from the original 12 A^3.  Hopefully it will be helpful for your debugging. 
```
### test
units           metal
boundary        p p p
atom_style      atomic
lattice         fcc 3.639087
region          box prism 0 50 0 20 0 2 0 0 0
create_box      1 box
create_atoms    1 box

pair_style      eam/fs
pair_coeff      * * ./CuZr_mm.eam.fs Cu
neighbor        1.0 bin
neigh_modify    delay 0 every 10 check yes

thermo          1000
thermo_style    custom step temp pxx pyy pzz pxy pxz pyz pe lx ly lz

variable        T equal 10.0
velocity        all create $T 4928459 rot yes dist gaussian
fix             1 all nvt temp $T $T 0.1
fix             2 all deform 100 xy erate 0.01 flip yes
compute         1 all voronoi/atom

shell           mkdir Shear
dump            1 all cfg 1000 ./Shear/cfg.* mass type xs ys zs c_1[1] c_1[2]
dump_modify     1 element Cu
run             150000
```

all the best,
Suzhi

**Comment by akohlmey** [comment]
Cannot be reproduced with 18 June 2016 version.
